examples/src/main/scala/org/apache/spark/examples/sql/hive/SparkHiveExample.scala

```diff
@@ -102,8 +101,63 @@ object SparkHiveExample {
     // |  4| val_4|  4| val_4|
     // |  5| val_5|  5| val_5|
     // ...
-    // $example off:spark_hive$
 
+    /*
```

Comment:
Oh just noticed this. You're using javadoc style comments here, but they won't have effect.
just use the `//` block style for comments that you see above, for consistency.

Reply:
+1

Reply:
@srowen Done, changes addressed